transition request draft (#40)
iherman/pmwg: admin/2026-06-epub-CR/transition-request.md

# Dependencies met (or not)
There were no major dependencies.

# Wide Review

For horizontal reviews:

- A11y: [link](https://github.com/w3c/a11y-request/issues/164) (originally 2026-03-30, re-opened 2026-05-27) [resolution (2026-06-24)](https://github.com/w3c/a11y-request/issues/164#issuecomment-4788314140)
- I18N: [link](https://github.com/w3c/i18n-request/issues/296) (2026-02-12) [resolution (2026-06-04)](https://github.com/w3c/i18n-request/issues/296#issuecomment-4622953684)
- Privacy: [link](https://github.com/w3cping/privacy-request/issues/209) (2026-04-10)[ resolution (2026-06-15)](https://github.com/w3cping/privacy-request/issues/209#issuecomment-4712364165)
- Security: [link](https://github.com/w3c/security-request/issues/130) (2026-04-10), _declaring timeout_
- TAG: [link](https://github.com/w3ctag/design-reviews/issues/1216) (2026-04-01) [resolution (2026-04-16)](https://github.com/w3ctag/design-reviews/issues/1216#issuecomment-4224209028)

> The horizontal reviews lead to a number of minor or editorial updates, see the aforementioned change logs.

Author: ```suggestion
The horizontal reviews led to a number of minor or editorial updates, see the aforementioned change logs.
```

> The horizontal reviews lead to a number of resulted minor or editorial updates, see the aforementioned change logs.

Reviewer: ```suggestion
The horizontal reviews lead to a number of minor or editorial updates, see the aforementioned change logs.
```
We need only one "lead to" or "resulted [in]"

Reviewer 2: And should be "led" not "lead"...

# Issues addressed

- https://github.com/w3c/epub-specs/issues?q=is%3Aissue%20label%3ASpec-EPUB3
- https://github.com/w3c/epub-specs/issues?q=is%3Aissue%20label%3ASpec-ReadingSystems
- https://github.com/w3c/epub-specs/issues?q=is%3Aissue%20label%3ASpec-Accessibility

> There are a number of open issues, mostly around multiple rendition and media overlays, that are considered as being beyond the current charter; that will be the topic of a new charter, currently under discussion.

Reviewer: ```suggestion
Several technical issues remain unresolved, primarily concerning multiple renditions and media overlays. These are beyond the current charter and will be the topic of a new charter, which is currently under discussion.
```
I'm concerned that "mostly around" could be ambiguous since "around" has so many meanings and connotations.

> There are a number of open issues, mostly around multiple rendition and media overlays, that are considered as being beyond the current charter; that will be the topic of a new charter, currently under discussion.

Reviewer 2: There's only https://github.com/w3c/epub-specs/issues/2806 that has anything to do with possible multiple renditions. I wouldn't think it warrants highlighting at the same scale as the media overlay issues.

Or did you mean something more like "alternative presentations with media overlays" to explain the various presentation models the changes could lead to?

Author: The comment became outdated by other merges, but I have added the sentence you proposed. Could look at it and, if needed, change it? Thx.

# Formal Objections

None.

# Implementation
[TODO: any preliminary implementation information? exit criteria? minimum CR duration?]

The exit criteria of the Working Group are described in:

- https://w3c.github.io/epub-specs/epub34/reports/exit_criteria.html

> with links to the report documents that will, eventually, contain the results of testing. There are a number of [tests](https://w3c.github.io/epub-tests/index.html) available (most of them inherited from EPUB 3.3, but new ones have been added for EPUB 3.4 features), new are being added.

Reviewer: ```suggestion
with links to the results of testing. There are a number of [tests](https://w3c.github.io/epub-tests/index.html) available. Most of them inherited from EPUB 3.3, and new ones have been added for EPUB 3.4 features.
```

> There are numerous EPUB 3 Reading System implementations on the market, some major ones are also represented on the Working Group (Google, Apple, Kobo, EDRLab, Colibrio, Kadokawa, Voyager, etc.). Many have already contributed tests results for EPUB 3.3 (which are, technically, valid though incomplete test results for EPUB 3.4). We expect most of these will contribute to the test results to include results on the new features.

Reviewer: ```suggestion
There are multiple current EPUB 3 reading system implementations. Developers represented within the Working Group include Apple, Colibrio, EDRLab, Google, Kadokawa, Kobo, and Voyager. Several of these organizations have contributed test results for EPUB 3.3. These existing results remain valid. However, additional testing is required to verify implementation of EPUB 3.4 features. The Working Group aims to secure testing contributions for these new features from all participating reading systems.
```
I alphabetized the list of RS to avoid any implication that we were setting priorities.  

Is "on the market" accurate? We support implementations that are not commercial, like specialized libraries and public school system implementations.

> There is a strong cooperation between the Working Group and the developers of [epubcheck](https://www.w3.org/publishing/epubcheck/), that already has a beta version of an EPUB 3.4 checker. This is particularly important, because epubcheck is used by virtually all publishers before releasing a new publication.

Reviewer: ```suggestion
There is a strong cooperation between the Working Group and the developers of [epubcheck](https://www.w3.org/publishing/epubcheck/). There is already a beta version of an EPUB 3.4 checker. This is particularly important, because epubcheck is used by virtually all publishers before releasing a new publication.
```

The minimal CR period goes until 2026-10-19.

# Patent disclosures

https://www.w3.org/groups/wg/epub/ipr

# Additional note on CR exit

The [Working Group's charter](https://www.w3.org/2020/12/epub3-wg-charter.html) has a very strong clause on backward compatibility:

> It is a primary goal of the new EPUB version to remain backward compatible with EPUB 3.3 (i.e., existing conformant EPUB 3.3 would remain conformant EPUB 3.4 documents).

> What this means that the usual CR approach, whereby if a feature does not get the right number of implementation per the exit criteria must be removed from the specification, cannot be applied in this case, except for features that did not exist prior to EPUB 3.3 (e.g., roll layout). Instead, the specification includes a separate section  on [obsolete features](https://www.w3.org/TR/epub-34/#app-obsolete).

Reviewer: ```suggestion
Consequently, the standard Candidate Recommendation (CR) process—which requires the removal of features that fail to meet the minimum implementation criteria—cannot be applied to legacy features. This exception does not apply to features introduced after EPUB 3.3 (for example, the roll layout type). To preserve backward compatibility, legacy features that lack the required number of implementations are retained and listed in a dedicated annex on [obsolete features](https://www.w3.org/TR/epub-34/#app-obsolete). 
```
